app/models/pokemon.rb

```diff
@@ -3,6 +3,8 @@ class Pokemon < ActiveRecord::Base
   belongs_to :species
   belongs_to :nature
 
+  default_scope -> { order('id ASC') }
```

Comment:
In general, you should avoid default scopes. Regardless, this could be written as `order(id: :asc)` instead